LinaSaysHello/Program.cs

```diff
@@ -1,28 +1,228 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.IO.Ports;
 using System.Linq;
+using System.Net.NetworkInformation;
+using System.Reflection;
+using System.Reflection.Emit;
 using System.Runtime.InteropServices;
 using System.Security.Cryptography;
 using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
+using System.Xml.Linq;
 
 namespace LinaSaysHello
 {
     internal class Program
     {
         static void Main(string[] args)
         {
-            //Console.WriteLine("Hi! What’s your name?");
-            //string name = Console.ReadLine();
-            //Console.WriteLine("Nice to meet you " + name);
-            //Console.WriteLine("Anastasiya");
-            //Console.WriteLine("Vention");
-            //Console.ReadLine();
+            //Task 1//
+            Console.Write("Enter any positive number: ");
+            int number = Convert.ToInt32(Console.ReadLine());
+            int sum = 0;
+            for (int i = 1; i <= number; i++)
+            {
+                sum += i;
 
-           
+            }
+            Console.WriteLine("Result: " + sum);
 
+            //Task2//
 
+            int number_3 = 3;
+            int counter = 0;
+
+            while (counter <= 10)
+            {
+                Console.WriteLine("3 x " + counter + " = " + (counter * number_3));
+                counter++;
+            }
+
+            //Task 3//
+
+            int[] numArray = { 3, 5, 9, 8, 15 };
+            int value = 1;
+            for (int i = 0; i < numArray.Length; i++)
+            {
+                value *=numArray[i];
+               
+            } Console.WriteLine(value);
+
+            // Task 4//
+
+            int divisionTimes = 0;
+            int result = 2048;
+
+            do
+            {
+                divisionTimes++;
+                result /= 2;
+                Console.WriteLine("Number after division: " + result);
+            }
+            while (result > 10);
+
+            Console.WriteLine(" It took " + divisionTimes + " times to divite 2048 by 2 till it's less than 10");
+
+            //  Task 5//
+            string[] words = { "yes", "Hello", "Other", "Words" };
+
+            for (int i = 0; i < words.Length; i++)
+            {
+                Console.WriteLine(words[i] + " ");
+
+            }
+            for (int i = 0; i < words.Length; i++)
+            {
+                if (words[i] == "Hello")
+                {
+                    Console.Write("\nLabas! ");
+                    continue;
```

Comment:
The **continue**  statement doesn't allow to leave the loop once needed word is found. So you need to use another statement)